Makefile

```diff
@@ -42,11 +47,11 @@ package-%:
 	bash scripts/copy_resources.sh $(PWD) $* && \
 	bash scripts/package.sh $(PWD) $*
 
-test-e2e-%: 
+test-e2e-%: build-%
 	@echo "[ test-e2e-%$* ]: Running e2e test..."
 	$(NEWRELIC_E2E) --commit_sha=test-string --retry_attempts=5 --retry_seconds=60 \
          --account_id=$(ACCOUNT_ID) --api_key=$(API_KEY) --license_key=$(LICENSE_KEY) \
-         --spec_path=$(PWD)/exporters/$*/e2e/e2e_spec.yml --verbose_mode=true
+         --spec_path=$(PWD)/exporters/$*/e2e/e2e_spec.yml --verbose_mode --agent_enabled=true
```

Comment:
agent enabled if I am not mistaken is true by default 
https://github.com/newrelic/newrelic-integration-e2e-action/blob/5613eb6b876cec56756f7f2f08472d8904c5752d/newrelic-integration-e2e/cmd/main.go#L30

Reply:
Updated here ad5be98927005d66ed4be4a68d9abcd2786124a2.